lib/get-html.ts

```diff
@@ -29,7 +29,14 @@ export const getHtml = async (
 
   // get only the HTML we care about to avoid unnecessary cache updates
   $('[data-testid="callout"]').remove();
-  const html = $("#article-contents").parent().parent().html() ?? "";
+  const data = $("#article-contents").parent().parent();
+  data.find("*").each((i, el) => {
+    const classes = $(el).attr("class") ?? "";
+    if (!classes.includes("language-json") && !classes.includes("warning")) {
+      $(el).removeAttr("class");
+    }
+  });
+  const html = data.html() ?? "";
```

Comment:
I think we are missing the use-case where an element has the className `language-json hsjfh` or `warning aksjdhksadj`, in these cases we would be having the same noise since we are not pruning thee hash for those HTMLElements.

I drafted something like this:

```suggestion
  const data = $("#article-contents").parent().parent();
  data.find("*").each((i, el) => {
    const classes = $(el).attr("class");

    if (classes) {
      console.log(classes);
      const filteredClasses = classes
        .split(" ")
        .filter((classSelector: string) =>
          ["language-json", "warning"].includes(classSelector)
        );

      if (filteredClasses.length) {
        $(el).attr("class", filteredClasses.join(" "));
      } else {
        $(el).removeAttr("class");
      }
    }
  });
  const html = data.html() ?? "";
```

Reply:
I just double checked the cache, and the `language-json` class is not present anymore. It seems that part of the code is out of date

Reply:
> I just double checked the cache, and the `language-json` class is not present anymore. It seems that part of the code is out of date

I would move this to a different PR. They are unrelated changes and in case we want to revert one thing does not imply we want to revert the other.

Reply:
All done